lib/SQS.js

```diff
@@ -370,10 +356,10 @@ function connectToQueue(queueUrl, callback) {
 				Name: queueUrl.split("/").pop(),
 				URL: queueUrl,
 				Created: new Date().toISOString(),
-				DelaySeconds: 0,
+				DelaySeconds: 10,
```

Comment:
@cwhenderson20 Yep, a relic from testing. I reverted the DelaySeconds and ReceiveMessageWaitTimeSeconds to zero and added a couple of tests